ui/src/app/kvm/components/CertificateMetadata/_index.scss

```diff
@@ -0,0 +1,14 @@
+@mixin CertificateMetadata {
+  .certificate-metadata {
+    @extend %vf-card;
```

Comment:
Could you just have used a `Card` here? If not it might be worth leaving a comment.

Reply:
I could have, and now do!